src/main/java/salesforce/entities/Account.java

```diff
@@ -9,12 +9,141 @@
 
 package salesforce.entities;
 
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.Supplier;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import salesforce.entities.constants.AccountConstant;
+
```

Comment:
imports are not necessary separate

please press ctrl + alt + L to the identation of the comments